scripts/workon.bat

```diff
@@ -71,4 +71,13 @@ if exist "%WORKON_HOME%\%VENV%\%VIRTUALENVWRAPPER_PROJECT_FILENAME%" (
     )
 )
 
+REM Run postactivate.bat

```

Comment:
The mother-project implements both local and global post-activate/deactivate hooks. This code implements the implements the global postactivate hook (only), while #73 implements the "local" post-activate/deactivate hooks. I would prefer a more comprehensive hook implementation PR...